IM_calculation/scripts/calculate_ims.py

```diff
@@ -60,7 +73,7 @@ def main():
         help="Please specify the version of the simulation. eg.18p4",
     )
     parser.add_argument(
-        "-m",
+        "-im",
```

Comment:
Does any script use the -m argument?

Reply:
this was a error while mering master to branch. fixing it.